rust/processor/src/models/default_models/parquet_transactions.rs

```diff
@@ -0,0 +1,390 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use super::{
+    block_metadata_transactions::BlockMetadataTransaction,
+    parquet_write_set_changes::{WriteSetChangeDetail, WriteSetChangeModel}, DataSize,
+};
+use crate::{
+    schema::transactions,
+    utils::{
+        counters::PROCESSOR_UNKNOWN_TYPE_COUNT,
+        util::{
+            get_clean_payload, get_clean_writeset, get_payload_type, standardize_address,
+        },
+    },
+};
+use ahash::AHashMap;
+use aptos_protos::transaction::v1::{
+    transaction::{TransactionType, TxnData},
+    Transaction as TransactionPB, TransactionInfo,
+};
+
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+use parquet_derive::{ParquetRecordWriter};
+
+#[derive(Clone, Debug, Deserialize, FieldCount, Identifiable, Serialize, ParquetRecordWriter)]
+#[diesel(primary_key(version))]
+#[diesel(table_name = transactions)]
+pub struct Transaction {
+    pub version: i64,
```

Comment:
pref:

- version -> txn_version
- txn_hash 
- block_height
- block_timestamp 
- epoch
- txn_type
- payload
- payload_type

rest is fine, are you sure `state_checkpoint_hash` is optional?

Reply:
![Screenshot 2024-05-31 at 2 18 07 PM](https://github.com/aptos-labs/aptos-indexer-processors/assets/38443641/73556194-f06c-4407-a007-603ae9866ab8)
It seems like it. 
<img width="1275" alt="Screenshot 2024-05-31 at 2 20 02 PM" src="https://github.com/aptos-labs/aptos-indexer-processors/assets/38443641/6530568a-f422-4106-99d9-68189f42be34">
state_change_hash seems required though